drivers/growatt/device.js

```diff
@@ -0,0 +1,40 @@
+// Copyright 2019: Rob, https://github.com/pro-sumer
+// License: GNU GPLv3, https://www.gnu.org/licenses/gpl.txt
+
+'use strict'
+
+const Inverter = require('../inverter')
+const api = require('./api')
+
+class Growatt extends Inverter {
```

Comment:
As username and password can be changed in settings, a check needs to occur before the settings change can proceed. Check `drivers/enphase/device.js` for an example (onInit, onSettings)

Reply:
I again used the Omnik class as an example. Fixed.